termwiz/src/escape/parser/mod.rs

```diff
@@ -173,6 +180,9 @@ impl<'a, F: FnMut(Action)> VTActor for Performer<'a, F> {
                 data: vec![],
             });
         } else {
+            if byte == b'p' && params == vec![1000] {
```

Comment:
I think we can avoid making a vec for this; the vec is undesirable because it represents an unconditional heap allocation just for a comparison.

```suggestion
            if byte == b'p' && params == [1000] {
```